build.gradle

```diff
@@ -40,7 +40,7 @@ def versions = [
         reformS2sClient    : '3.0.0',
         serenity           : '2.0.23',
         sonarPitest        : '0.5',
-        springBoot         : '2.3.1.RELEASE',
+        springBoot         : '2.2.6.RELEASE',
         springHystrix      : '2.1.1.RELEASE',
```

Comment:
do we need to revert spring boot version to 2.2.6?
Ideally we should keep it at 2.3.1

Reply:
reverted version